code/controllers/subsystem/chat.dm

```diff
@@ -20,16 +20,16 @@ SUBSYSTEM_DEF(chat)
 	var/list/client_to_sequence_number = list()
 
 /datum/controller/subsystem/chat/proc/generate_payload(client/target, message_data)
-	var/sequence = client_to_sequence_number[target.ckey]
-	client_to_sequence_number[target.ckey] += 1
+	var/sequence = client_to_sequence_number[target.ckey]++
```

Comment:
how much time does this actually save compared to the in list thing

Reply:
Doubt it saves much, i did it because it turns two associated lookups into 1.

Reply:
I really dislike this syntax, are you open to changing it?